core/server/src/test/java/alluxio/worker/block/BlockWorkerTest.java

```diff
@@ -106,10 +108,11 @@ public void before() throws IOException {
   }
 
   /**
-   * Resets the worker context.
+   * Resets the worker context and close the data server on clean up.
    */
   @After
-  public void after() {
+  public void after() throws IOException {
+    ((DataServer) Whitebox.getInternalState(mBlockWorker, "mDataServer")).close();
```

Comment:
Oh my bad, this should be removed (as BlockWorker does not hold the data server anymore)